### PR TITLE
[BOP-1018] add docs for coredns lameduck

### PR DESCRIPTION
**Pull request by sakshisharma84**
## Description

Add docs for coredns lameduck feature

## JIRA Ticket
https://mirantis.jira.com/browse/BOP-1018

### CODE REVIEW

File: content/docs/operations/coredns-lameduck.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The lameduck parameters that you can configure are
+detailed in the following table.
+
+| Field                      | Description                                                             | Default |
+|----------------------------|-------------------------------------------------------------------------|---------|
+| enabled                    | Enables the lameduck health function.<br/>  Valid values: true, false.  | false   |
+| duration                   | Length of time during which lameduck will run, expessed with integers and time suffixes, such as s for seconds and m for minutes.                                            | 7s      |
+
+    Note: Editing the CoreDNS config map outside MKE to configure the lameduck function is not supported. Any such attempts shall be superseded by the values that are configured in the MKE configuration file.
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:30]:
```suggestion
{{< callout type="info" >}}Editing the CoreDNS config map outside MKE to configure the lameduck function is not supported. Any such attempts will be superseded by the values that are configured in the MKE configuration file. {{< /callout >}}
```

File: content/docs/operations/coredns-lameduck.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```yaml
+  dns:
+    lameduck:
+      enabled: true
+      duration: "7s"
+```
+
+The lameduck parameters that you can configure are
+detailed in the following table.
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:23]:
```suggestion
**Configuration parameters** 
```

File: content/docs/operations/coredns-lameduck.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+| Field                      | Description                                                             | Default |
+|----------------------------|-------------------------------------------------------------------------|---------|
+| enabled                    | Enables the lameduck health function.<br/>  Valid values: true, false.  | false   |
+| duration                   | Length of time during which lameduck will run, expessed with integers and time suffixes, such as s for seconds and m for minutes.                                            | 7s      |
+
+    Note: Editing the CoreDNS config map outside MKE to configure the lameduck function is not supported. Any such attempts shall be superseded by the values that are configured in the MKE configuration file.
+
+
+## Applying the Configuration
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:33]:
```suggestion
## Apply configuration
```

File: content/docs/operations/coredns-lameduck.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+[INFO] plugin/reload: Running configuration SHA512 = 591cf328cccc12bc490481273e738df59329c62c0b729d94e8b61db9961c2fa5f046dd37f1cf888b953814040d180f52594972691cd6ff41be96639138a43908
+CoreDNS-1.10.1
+linux/arm64, go1.20, 055b2c3
+
+[INFO] Reloading
+[INFO] plugin/reload: Running configuration SHA512 = 26fe33ee13757f04c8c9a1caebd7c6f0614306c92089ea215f1a8663f95ff1e673d4fa5de544b31492231923d4679370ce8735823ce3b5e65e5c23a9029c4512
+[INFO] Reloading complete
+```
+
+## MKE version comparison: Lameduck configuration parameters
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:55]:
```suggestion
## MKE version comparison

 **Lameduck configuration parameters**
```

File: content/docs/operations/coredns-lameduck.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+|----------------------------|-------------------------------------------------------------------------|---------|
+| enabled                    | Enables the lameduck health function.<br/>  Valid values: true, false.  | false   |
+| duration                   | Length of time during which lameduck will run, expessed with integers and time suffixes, such as s for seconds and m for minutes.                                            | 7s      |
+
+    Note: Editing the CoreDNS config map outside MKE to configure the lameduck function is not supported. Any such attempts shall be superseded by the values that are configured in the MKE configuration file.
+
+
+## Applying the Configuration
+
+After enabling/adjusting the lameduck configuration, you must wait for the CoreDNS pods to apply the changes. You can check the CoreDNS logs to verify that the lameduck configuration is applied.
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:35]:
```suggestion
1. Enable or adjust the lameduck configuration. 
2. Wait for the CoreDNS pods to apply the changes. 
3. Check the CoreDNS logs to verify if the lameduck configuration is applied:
```

File: content/docs/operations/coredns-lameduck.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+linux/arm64, go1.20, 055b2c3
+
+[INFO] Reloading
+[INFO] plugin/reload: Running configuration SHA512 = 26fe33ee13757f04c8c9a1caebd7c6f0614306c92089ea215f1a8663f95ff1e673d4fa5de544b31492231923d4679370ce8735823ce3b5e65e5c23a9029c4512
+[INFO] Reloading complete
+```
+
+## MKE version comparison: Lameduck configuration parameters
+
+| MKE-3                                                 | MKE-4                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (MagdaDziadosz) [inline content/docs/operations/coredns-lameduck.md:57]:
```suggestion
| MKE 3                                                 | MKE 4                                                                                                                                                                                                                                                                                                                                                                                                                |
```

**Comment by sakshisharma84** [comment]
@MagdaDziadosz thanks for the review. All comments incorporated.

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**